fearless/Modules/Crowdloan/CrowdloanList/CrowdloanListViewController.swift

```diff
@@ -0,0 +1,274 @@
+                titleCell.bind(title: R.string.localizable.crowdloanListSectionFormat(symbol))
+                return titleCell
+            case 1:
+                let yourCrowdloansCell = tableView.dequeueReusableCellWithType(YourCrowdloansTableViewCell.self)!
+                let counter = viewModel.contributionsCount ?? ""
+                yourCrowdloansCell.bind(details: counter, for: selectedLocale)
+                return yourCrowdloansCell
+            default:
+                return UITableViewCell()
+            }
+        } else if indexPath.section == 1, let active = viewModel.active {
+            return createActiveTableViewCell(
+                tableView,
+                viewModel: active.crowdloans[indexPath.row].content
+            )
+        }
+
+        if let completed = viewModel.completed {
+            return createCompletedTableViewCell(
+                tableView,
+                viewModel: completed.crowdloans[indexPath.row].content
+            )
+        }
+
+        return UITableViewCell()
+    }
+
+    private func createActiveTableViewCell(
+        _ tableView: UITableView,
+        viewModel: ActiveCrowdloanViewModel
+    ) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCellWithType(ActiveCrowdloanTableViewCell.self)!
+        cell.bind(viewModel: viewModel)
+        return cell
+    }
+
+    private func createCompletedTableViewCell(
+        _ tableView: UITableView,
+        viewModel: CompletedCrowdloanViewModel
+    ) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCellWithType(CompletedCrowdloanTableViewCell.self)!
+        cell.bind(viewModel: viewModel)
+        return cell
+    }
+}
+
+extension CrowdloanListViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: true)
+    }
+
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        guard section > 0, case let .loaded(viewModel) = state else {
+            return nil
+        }
+
+        let headerView: CrowdloanStatusSectionView = tableView.dequeueReusableHeaderFooterView()
+
+        if section == 1, let active = viewModel.active {
+            headerView.bind(title: active.title, status: .active)
+        } else if let completed = viewModel.completed {
+            headerView.bind(title: completed.title, status: .completed)
+        }
+
+        return headerView
+    }
+
+    func tableView(_: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        section > 0 ? 40.0 : 0.0
+    }
+}
+
+extension CrowdloanListViewController: CrowdloanListViewProtocol {
+    func didReceive(state: CrowdloanListState) {
+        self.state = state
+
+        applyState()
+    }
+}
+
+extension CrowdloanListViewController: Localizable {
+    func applyLocalization() {
+        if isViewLoaded {
+            setupLocalization()
+        }
```

Comment:
tableView.reloadData()?

Reply:
As you can see I added localization manager also to presenter. I thought it has several benefits: 
1) Presenter decides whether model depends on localization and reloads view if needed, an view just decides localization for a static content;
2) We don't have overhead with ```LocalizableResource``` stuff anymore.